distribution/tools/keystore-cli/build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
   compileOnly project(":server")
   compileOnly project(":libs:opensearch-cli")
   testImplementation project(":test:framework")
-  testImplementation 'com.google.jimfs:jimfs:1.2'
+  testImplementation 'com.google.jimfs:jimfs:1.3'
```

Comment:
can you please replace `testImplementation 'com.google.jimfs:jimfs:1.3`' with 
`testImplementation 'com.google.jimfs:jimfs:1.3.0'` here.

It is throwing error that it is not able to pull anything from `https://repo.maven.apache.org/maven2/com/google/jimfs/jimfs/1.3/jimfs-1.3.pom` instead we should pull from `https://repo.maven.apache.org/maven2/com/google/jimfs/jimfs/1.3.0/jimfs-1.3.0.pom`

Reply:
Similarly replace bumping 1.3 with bumping 1.3.0 in other file changes as well.

Reply:
Will do